### PR TITLE
image plugin styling and onPaste

### PR DESCRIPTION
**Pull request by Knorrke**
## Added

- Added Checkbox, Button and Textarea components for overlay
- Added onPaste handler for image plugin accepting jpg, png, bmp, gif, svg

grafik

### CODE REVIEW

**Comment by Knorrke** [comment]
ping @inyono can you look at the failing deploy checks?

**Comment by inyono** [comment]
Should work if you rebase on master :) (since the build-storybook task isn't in your branch yet)

File: packages/plugin-image/src/index.ts [hunk 1]
```diff
@@ -15,7 +15,22 @@ export const createImagePlugin = <T = unknown>(
 ): StatefulPlugin<typeof imageState> => {
   return {
     Component: createImageEditor(config),
-    state: imageState
+    state: imageState,
+    onPaste: (clipboardData: DataTransfer) => {
+      const value = clipboardData.getData('text')
+
+      if (/(.*)\.(jpe?g|png|bmp|gif|svg)$/.test(value)) {
```
Comment (inyono) [inline packages/plugin-image/src/index.ts]:
```suggestion
      if (/\.(jpe?g|png|bmp|gif|svg)$/.test(value)) {
```

**Comment by Knorrke** [comment]
alright, build worked now :-)

**Comment by Atyanar** [comment]
I like this, too. Code is working for me.

In the long run I would probably opt for the image sitting centered by default (currently its always on the left). But this is a nice improvement! :)

File: packages/ui/src/elements/textarea.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import { ThemeProps } from 'styled-components'
+
+const TextareaLabel = styled.label((props: ThemeProps<EditorTheming>) => ({
+  color: props.theme.textColor,
+  display: 'flex',
+  flexDirection: 'row',
+  margin: '20px auto 0px',
+  justifyContent: 'space-between'
+}))
+TextareaLabel.defaultProps = { theme: defaultTheming }
```
Comment (inyono) [inline packages/ui/src/elements/textarea.tsx]: Do we really need that in every component accessing the theme?

**Pull request merged**